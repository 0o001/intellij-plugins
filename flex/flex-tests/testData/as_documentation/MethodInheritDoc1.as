package {

public class Sub extends Base {

  /**
   * (sub 1)
   * (sub 2)
   * @inheritDoc 
   * (sub 3)
   */
  public override function f<caret>oo(p:String, v:int) : Boolean {

  }
}
}
