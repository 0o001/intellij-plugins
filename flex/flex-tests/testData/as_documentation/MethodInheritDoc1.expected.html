<div class='definition'><pre>Sub.foo(p: String,<br>        v: int): Boolean</pre></div><div class='content'><p>(sub 1)
<p>(sub 2)
<p>(sub 3)
<p>(base 1)
<p>(base 2)
<p>(base 3)
<p>(grandbase1)
<p>(grandgrandbase1)</div><table class='sections'><tr><td valign='top' class='section'><p>Params:</td><td valign='top'><p>p &ndash; (base p)<p>v &ndash; (grandbase v)<tr><td valign='top' class='section'><p>Returns:</td><td valign='top'><p>(fake) (base return)</td></table>